adafruit_rockblock.py

```diff
@@ -190,7 +190,15 @@ def status(self):
             status = resp[1].strip().decode().split(":")[1]
             return tuple([int(a) for a in status.split(",")])
         return (None,) * 6
-
+      
+    @property
+    def signalStrength(self):
```

Comment:
Rename to `signal_quality` to avoid use of camel case and better match parameter name to ISU AT Command Reference.